File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/recomendaciones/interfaces/recurso.ts
import { RecursoDet } from './recurso-det';
export interface Recurso {
  id_recurso: number;
  id_recomendacion: number;
  codigo: string;
  descripcion: string;
  icono: string;
  enlace_ruta: string;
  hashtag: string;

  detalle: RecursoDet[]
}

File: Prod.RutaDigital.Entidades/Recurso/Recurso.cs
namespace Prod.RutaDigital.Entidades;

public class Recurso
{
    public int id_recurso { get; set; }
    public int id_recomendacion { get; set; }
    public string? codigo { get; set; }
    public string? descripcion { get; set; }
    public string? icono { get; set; }
    public string? enlace_ruta { get; set; }
    public string? red_social { get; set; }
    public string? hashtag { get; set; }
}
